distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc

```diff
@@ -83,62 +84,68 @@
 |Size in bytes of resource.
 |String
-|
+|Although this type cannot be changed for legacy reasons, its value should always be a parseable whole number
 |
 
 |thumbnail
 |The thumbnail for the resource in JPEG format.
-|Binary
+|Base 64 encoded binary string per RFC 4648
 |<= 128 KB
 |
 
 |description
-|An account of the resource. http://dublincore.org/documents/dcmi-terms/#elements-description[Dublin Core elements-description]
+|An account of the resource. http://dublincore.org/documents/dcmi-terms/#elements-description[Dublin Core elements-description].
 |String
 |
 |
 
 |checksum
-|Checksum value for the primary resource for the metacard
+|Checksum value for the primary resource for the metacard.
 |String
 |< 1024 characters
 |
 
 |checksum-algorithm
-|Algorithm used to calculate the checksum on the primary resource of the metacard
+|Algorithm used to calculate the checksum on the primary resource of the metacard.
 |String
 |< 1024 characters
 |
 
 |created
-|The creation date of the resource http://dublincore.org/documents/dcmi-terms/#terms-created[Dublin Core terms-created]
+|The creation date of the resource http://dublincore.org/documents/dcmi-terms/#terms-created[Dublin Core terms-created].
 |Date
 |
 |
 
 
 |modified
-|The modification date of the resource http://dublincore.org/documents/dcmi-terms/#terms-modified[Dublin Core terms-modified]
+|The modification date of the resource http://dublincore.org/documents/dcmi-terms/#terms-modified[Dublin Core terms-modified].
 |Date
 |
 |
 
 |language
-|The language of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#language[Dublin Core language]
-|String
-|Alpha-3 language code per ISO_639-2
+|The language(s) of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#language[Dublin Core language].
+|List of Strings
+|Alpha-3 language code(s) per ISO_639-2
 |
 
-|resource-derived-download-url
-|Download URL for accessing the derived formats for the metacard resource.
-|String
-|Valid URL per RFC 2396
+|resource.derived-download-url
+|Download URL(s) for accessing the derived formats for the metacard resource.
+|List of Strings
+|Valid URL(s) per RFC 2396
+|
+
+|resource.derived-uri
+|Location(s) for accessing the derived formats for the metacard resource.
+|List of Strings
+|Valid URI per RFC 2396
 |
 
 |datatype
-|The generic type of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#terms-type[Dublin Core terms-type] and extended to include other DDF supported types.
-|String
-|Collection, Dataset, Event, Image, Interactive Resource, Service, Software, Sound, Text, Video, Document
+|The generic type(s) of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#terms-type[Dublin Core terms-type] and extended to include other DDF supported types.
+|List of Strings
+|One of Collection, Dataset, Event, Image, Interactive Resource, Service, Software, Sound, Text, Video, Document
```

Comment:
Probably was there before we realized it was a list and not just a single value. Probably more suited as "One or more of"